cmd/context/create.go

```diff
@@ -282,6 +283,33 @@ func (c *Command) initOktetoContext(ctx context.Context, ctxOptions *Options) er
 	okteto.GetContext().IsTrial = clusterMetadata.IsTrialLicense
 	okteto.GetContext().CompanyName = clusterMetadata.CompanyName
 
+	if len(clusterMetadata.CliVersionRange) > 0 {
```

Comment:
I was testing with a local build of this branch, and with the cluster updated with this feature where I've set that I can only use the latest CLI. Should I get some warning in that case considering that the version is the commit?

I was adding some logs to see if the version that I set in the backend matches the verison that the CLI receives, simply adding `fmt.Println("CliVersionRange", clusterMetadata.CliVersionRange)`, but I always get an empty array. I've checked the query and looks good. Can you help me spot why it's logging an empty array? What am I missing?